src/amo/components/AddonAuthorLinks/index.js

```diff
@@ -0,0 +1,72 @@
+/* @flow */
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import type { AddonType } from 'core/types/addons';
+import DefinitionList, { Definition } from 'ui/components/DefinitionList';
+import type { I18nType } from 'core/types/i18n';
+import type { AppState } from 'amo/store';
+import { isAddonAuthor } from 'core/utils';
+
+type Props = {|
+  addon: AddonType | null,
+|};
+
+type InternalProps = {|
+  ...Props,
+  i18n: I18nType,
+  userId: number | null,
```

Comment:
it should probably be `currentUserID`

Reply:
On changing the variable name to `currentUserID` the code breaks for some weird reason.